src/listeners/dad.ts

```diff
@@ -0,0 +1,25 @@
+import {Listener} from "@ubccpsc310/bot-base";
+import {Client, Message} from "discord.js";
+
+const dad: Listener<"messageCreate"> = {
+    event: "messageCreate",
+    procedure: async (client: Client, message: Message) => {
+        if (message.author.bot) return;
+
+        const messageSendersActualName = getMessageSendersActualName(message.cleanContent);
+        if (messageSendersActualName == null) return; // u have no name
```

Comment:
I hate so much that this works.

Reply:
i only just saw this lmao, is there a better way you wouldve done it? if youre talking about the backwards if statements, i tend to feel that less nesting is usually more readable, but idk lmao